Phase01Project1/src/Documents/Document.java

```diff
@@ -0,0 +1,35 @@
+package Documents;
+
+public abstract class Document{
+    private String name;
+    private String text;
```

Comment:
this abstract class (or interface) should have a method named, getText. I think any document has a different way of initialization and what we want from a document is just a text. This change helps open/closed (O of SOLID).